packages/kit/src/cli.js

```diff
@@ -298,10 +295,7 @@ export async function get_vite_config(svelte_config) {
 		}
 	}
 
-	const { sveltekit } = await import('./vite/index.js');
-
-	// TODO: stop reading Vite config from SvelteKit config or move to CLI
-	const vite_config = await svelte_config.kit.vite();
-	vite_config.plugins = [...(vite_config.plugins || []), ...sveltekit()];
-	return vite_config;
+	throw new Error(
+		'vite.config.js is missing. See an example here: https://github.com/sveltejs/kit/blob/master/packages/create-svelte/templates/skeleton/vite.config.js'
+	);
```

Comment:
I think we should print a sample config, like we do for tsconfig.json and wrangler.toml etc

```suggestion
	console.error(colors.bold().red('> vite.config.js is required'));
	console.error(
		`
		Sample vite.config.js:

		import { sveltekit } from '@sveltejs/kit/vite';

		/** @type {import('vite').UserConfig} */
		const config = {
			plugins: [sveltekit()]
		};

		export default config;

		`.replace(/^\t{2}/gm, '')
	);

	process.exit(1);
```